docs/_documentations/troubleshooting.md

```diff
@@ -504,7 +504,25 @@ Issue link: https://github.com/eclipse/codewind-docs/issues/64
 ## Node.js and Swift templates remain in the starting state
 The templates `Appsody Node.js template` and `Appsody Swift template` remain in the starting state by default because these templates do not have a server implemented, and therefore, its status cannot be detected. These templates do not have a server and are intended to help you implement your own server.
```

Comment:
Change tenses:

`these templates do not have a server implemented, and therefore, its status cannot be detected.` should be changed to:
`these templates do not have a server implemented, and therefore, their statuses cannot be detected.`

There are two templates, so its->their, and since they each have their own undetected status, we should indicate such: status->statuses.

Reply:
ok can fix